### PR TITLE
Added doc string in missing files

### PR DESCRIPTION
**Pull request by vineetpalan**

### CODE REVIEW

**Comment by vineetpalan** [comment]
@nirzari Can you review it.

File: reviewrot/basereview.py [hunk 1]
```diff
@@ -195,15 +210,15 @@ def format_duration(created_at):
...
-        """ Format the result in a given style.
-
-        style - the name of the style.
-        i - an integer, indicating the position in a list.
-        N - the total size of the list.
+        """
+        Format the result in a given style.
+        Args:
+            style(str): the name of the style.
+            i(int): an integer, indicating the position in a list.
```
Comment (nirzari) [inline reviewrot/basereview.py]: You could just write, 'position in a list' or 'index' or something simpler and not mention data type again in desc

File: reviewrot/basereview.py [hunk 2]
```diff
@@ -195,15 +210,15 @@ def format_duration(created_at):
...
+            N(int): The total size of the list.
```
Comment (nirzari) [inline reviewrot/basereview.py]: length of the list might be a better term here.

File: reviewrot/basereview.py [hunk 3]
```diff
@@ -213,6 +228,14 @@ def format(self, style, i, N):
         return lookup[style](i, N)
 
     def _format_oneline(self, i, N):
+        """
+        Format the result in oneline style.
+        Args:
+            i(int): an integer, indicating the position in a list.
```
Comment (nirzari) [inline reviewrot/basereview.py]: same as above

File: reviewrot/basereview.py [hunk 4]
```diff
@@ -213,6 +228,14 @@ def format(self, style, i, N):
...
+            N(int): the total size of the list.
```
Comment (nirzari) [inline reviewrot/basereview.py]: same as baove

File: reviewrot/basereview.py (hunk 3 above)
Comment (nirzari) [inline reviewrot/basereview.py]: Please change this in remaining script as well

File: reviewrot/basereview.py [hunk 5]
```diff
@@ -269,7 +273,7 @@ def _format_json(self, i, N):
         """
         Format the result in json style.
         Args:
-            i(int): an integer, indicating the position in a list.
+            i(int): position in a list.
```
Comment (nirzari) [inline reviewrot/basereview.py]: May be length of the list?

**Pull request merged**